client-side/src/components/Report/htmlPDF.jsx

```diff
@@ -0,0 +1,28 @@
+import html2canvas from 'html2canvas';
+import jsPDF from 'jspdf';
+import React from 'react';
+import GenericButton from '../../stories/Button/GenericButton';
+
+const DownloadPage = ({divID}) => {
+  const downloadPDF = () => {
+    const input = document.getElementById(divID);
+    html2canvas(input, { useCORS: true }).then((canvas) => {
+      const imgData = canvas.toDataURL('image/png');
+      const pdf = new jsPDF('p', 'mm', 'a4');
```

Comment:
what are those variables? (p, mm, a4)
maybe they should come from props and those are the default values?